express-app/app.models.js

```diff
@@ -15,3 +15,19 @@ exports.fetchArticleID = (articleID) => {
     return rows[0];
   });
 };
+
+exports.fetchArticles = (sort_by, order) => {
+  let query = `SELECT articles.*, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id`;
+
+  if (sort_by) {
+    query += ` ORDER BY ${sort_by}`;
+    query += ` ${order.toUpperCase()}`;
+  }
+
+  return db.query(`${query};`).then(({ rows }) => {
+    return rows.map((article) => {
+      const { body, ...removedBody } = article;
+      return removedBody;
+    });
```

Comment:
This does work, but it is not very efficient. Ideally you want the query to do this instead of having to iterate using JS over the array and removing the body property, which if given a large data set could result in a slower response